README.md

```diff
@@ -11,3 +11,8 @@ Employee Impotance(https://leetcode.com/problems/employee-importance/)
 
 
 
+//problem1. java has O(m*n) time complexity and O(m*n) space compelexity
+
+//problem2B. java has O(n) time complexity and O(n) space compelexity
+
+//problem2D. java has O(n) time complexity and O(n) space compelexity
```

Comment:
Please specify what `m` and `n` indicate.

Reply:
updated.